packages/app-123code/src/SummaryBar.tsx

```diff
@@ -15,7 +14,7 @@ import { formatBalance, formatNumber } from '@polkadot/util';
 
 import translate from './translate';
 
-interface Props extends ApiProps, BareProps, I18nProps {
+interface Props extends BareProps, I18nProps {
```

Comment:
Technically `withCalls` stuff adds the `ApiProps` stuff, but since we are not using it, all ok.